cmd/thanos/query-frontend.go

```diff
@@ -16,93 +16,74 @@ import (
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/common/model"
-	"github.com/thanos-io/thanos/pkg/extkingpin"
 	"github.com/weaveworks/common/user"
+	"gopkg.in/yaml.v2"
 
+	"github.com/cortexproject/cortex/pkg/util/validation"
```

Comment:
One small nit. Can you please move this import above?

Reply:
updated